react/components/EXPERIMENTAL_Table/DataTable/RowContainer.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FC } from 'react'
+
+import useTableContext from '../hooks/useTableContext'
+
+const RowContainer: FC<RowContainerProps> = ({ id, children }) => {
```

Comment:
Is it reused anywhere else? I'm curious why this is a component and not just a div used in `Row.tsx`

Reply:
It is used both on `Row` and `RowNode`

Reply:
Changed it